example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
```

Comment:
Theoretisch könnte man noch höher gehen, oder?
In der München App haben wir im Gradle Wrapper aktuell gradle-7.3.3

Reply:
Ui, ich konnte sogar schon auf 7.4 upgradeln :)